src/helpers/positionHelper.ts

```diff
@@ -0,0 +1,55 @@
+type LatLon = {
+  latitude: number;
+  longitude: number;
+};
+
+const calculateDistance = (a: LatLon, b: LatLon) =>
+  Math.sqrt(Math.pow(a.latitude - b.latitude, 2) + Math.pow(a.longitude - b.longitude, 2));
```

Comment:
do you have a source for this calculation?

Reply:
no, this is just rectangular approximation.
but thanks for pointing it out.

due to us being relatively far north, I added a factor of ~0.63 (north of germany would be ~0.58, south of germany 0.68) to the longitude values (see https://www.movable-type.co.uk/scripts/latlong.html -> equirectangular approximation)
if we ever want to use this feature globally (not restricted to germany) then we need to revisit this.
this is a decent enough approximation for our use case though.

I added a reference and explanation of the calculation to the code